<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptDialog</class>
 <widget class="QDialog" name="ScriptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/script_pencil.png</normaloff>:/img/script_pencil.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_script_name">
       <property name="text">
        <string>Script Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_script_name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_main">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter_script">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QPlainTextEdit" name="script_edit">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
      </widget>
      <widget class="QTextEdit" name="txt_script_log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabInfo">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabHelp">
       <attribute name="title">
        <string>Help</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="text_help">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAttTraits">
       <attribute name="title">
        <string>Attributes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter_attributes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_att_info">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_attributes">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHealth">
       <attribute name="title">
        <string>Health</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QSplitter" name="splitter_health">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_health_info">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_health">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabItems">
       <attribute name="title">
        <string>Equipment</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QSplitter" name="splitter_equip">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_item_info">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_items">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPersonality">
       <attribute name="title">
        <string>Personality</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QSplitter" name="splitter_personality">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_personality_info">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_personality">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabJobs">
       <attribute name="title">
        <string>Profession &amp;&amp; Job</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter_job">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_job_info">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_jobs">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLabors">
       <attribute name="title">
        <string>Labors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter_labors">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_labor_info">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_labors">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSkills">
       <attribute name="title">
        <string>Skills</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_skills">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="txt_skill_info">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="text_skills">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_save_status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="toolTip">
        <string>Tests the current script, but doesn't save it. This is tested in addition to the current filters or scripts already applied!</string>
       </property>
       <property name="text">
        <string>Test Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="toolTip">
        <string>Saves the current script.</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>ScriptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply_script()</slot>
  <slot>save_script()</slot>
 </slots>
</ui>
